smartseq2_single_sample/tasks/picard.wdl

```diff
@@ -0,0 +1,56 @@
+task CollectMultipleMetrics {
+  File aligned_bam
+  File ref_genome_fasta
+  String output_filename
+  
+  command {
+    java -Xmx4g -jar picard.jar CollectMultipleMetrics \
+      VALIDATION_STRINGENCY=SILENT \
+      METRIC_ACCUMULATION_LEVEL=ALL_READS \
+      INPUT=${aligned_bam} \
+      OUTPUT="${output_filename}" \
+      FILE_EXTENSION = ".txt" \
+      PROGRAM = CollectAlignmentSummaryMetrics \
+      PROGRAM = CollectInsertSizeMetrics \
+      PROGRAM = CollectGcBiasMetrics \
+      PROGRAM = RnaSeqMetrics \
+      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      ASSUME_SORTED=true
```

Comment:
Will this always be true? If not this should be a variable

Reply:
For us, it is always be true.